modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -624,6 +619,11 @@
 	<td>//label[@for='type']/..//select[@id='type']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PROPERTIES_TYPE_TEXTAREA_BY_POSITION</td>
+	<td>xpath=(//label[@for='type']/..//select[@id='type'])[${key_sectionOption}]</td>
```

Comment:
Can you check if it's possible to use `key_index` instead of `key_sectionOption` without much work? If this requires a refactor of the macro/testcase, you can leave as it is, as it's not the goal of this ticket.